src/frontend/src/components/plugins/RemoteComponent.tsx

```diff
@@ -57,10 +62,23 @@ export default function RemoteComponent({
         .then((func) => {
           if (func) {
             try {
-              func(componentRef.current, context);
+              // Render the plugin component into the target element
+              // Note that we have to provide the right context(s) to the component
+              createRoot(componentRef.current!).render(
+                <ApiProvider client={queryClient} api={api}>
+                  <MantineProvider
+                    theme={context.theme}
+                    colorSchemeManager={colorSchema}
+                  >
+                    <LanguageContext>{func(context)}</LanguageContext>
```

Comment:
Does this mean, we can't use any non-react plugins anymore? I would really not use this. Can't we just pass the createcontext function to the plugin? And also doesn't this break existing plugins?

Reply:
Will look into passing this through to the plugin.

Reply:
@wolflu05 ok, I have patched it to support the "old" way of passing context through to the plugin

However, the *new* way is MUCH better and is required to allow any of these new features to work